html2markdown.py

```diff
@@ -180,13 +180,15 @@ def _markdownify(tag, _listType=None, _blockQuote=False, _listIndex=1):
 			# process children first
 			for child in children:
 				_markdownify(child)
-			if tag.string != tag['href'] or tag.has_attr('title'):
+			if tag.string != tag.get('href') or tag.has_attr('title'):
```

Comment:
It should probably leave an anchor tag unchanged if it doesn't have an href attribute instead. The old hack in the else clause can be removed.